taichi/transforms/offload.cpp

```diff
@@ -14,20 +14,18 @@ namespace {
 // Offloaded local variables to its offset in the global tmps memory.
 using StmtToOffsetMap = std::unordered_map<const Stmt *, std::size_t>;
 
-std::unique_ptr<std::unordered_map<OffloadedStmt *, Stmt *>> begin_stmt,
-    end_stmt;
+struct OffloadedToBeginEndStmts {
```

Comment:
Name nit: why `to`?
```suggestion
struct OffloadedRanges {
```

Reply:
Nice! Since many places need to be changed, I will make the change locally.